app/assets/javascripts/applicants/admissions_admin_email.js

```diff
@@ -0,0 +1,12 @@
+$(function() {
+    $('input').keyup(function (event) {
+        var intro = $('input[name="introduction"]').val()
+        var subj = $('input[name="subject"]').val()
+        $("#preview-subject").text(subj)
+        $("#preview-intro").text(intro+" Pernille,")
```

Comment:
Should probably change the default name to something like `<Name>`.

Reply:
True, this was really just a silly easter egg for the current admission leader. Should probably do this to make it clear that the name is automatically set!

Reply:
Ah, didn't realize it was an easter egg 😅 That changes things!